.github/workflows/full-unittests.yml

```diff
@@ -0,0 +1,91 @@
+  workflow_dispatch: # used for debugging or manual validation of a branch
+
+jobs:
+  py_unit_tests:
+    runs-on: single-cell-1tb-runner
+    strategy:
+      fail-fast: false # prevent this job from killing other jobs
+    steps:
+      - name: log system state
+        run: |
+          free
+          echo ---------
+          df -kh
+          echo ---------
+          lscpu
+
+      - name: install OS dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y build-essential git-all libxml2-dev libssl-dev libcurl4-openssl-dev cmake
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: install python dependencies (including experimental)
+        run: |
+          python -m pip install -U pip setuptools setuptools_scm wheel
+          pip install --use-pep517 accumulation-tree # Geneformer dependency needs --use-pep517 for Cython
+          pip install -r ./api/python/cellxgene_census/scripts/requirements-dev.txt
+          pip install './api/python/cellxgene_census/[experimental]'
+          echo '--------- tiledbsoma.show_package_version():'
+          python -c 'import tiledbsoma; tiledbsoma.show_package_versions()'
+          echo '--------- PIP package versions:'
+          pip freeze
+
+      - name: pytest (--expensive --experimental)
+        run: |
+          PYTHONPATH=. pytest -v --durations=0 -rP --experimental --expensive ./api/python/cellxgene_census/tests/
+
+  r_unit_tests:
+    runs-on: single-cell-1tb-runner
+    strategy:
+      fail-fast: false # prevent this job from killing other jobs
+    steps:
+      - name: log system state
+        run: |
+          free
+          echo ---------
+          df -kh
+          echo ---------
+          lscpu
+
+      - name: install OS dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y build-essential git-all libxml2-dev libssl-dev libcurl4-openssl-dev cmake
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          extra-repositories: https://tiledb-inc.r-universe.dev, https://cloud.r-project.org, https://chanzuckerberg.r-universe.dev
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          working-directory: ./api/r/cellxgene.census
+          extra-packages: any::rcmdcheck, any::remotes
+          cache: true
+
+      - name: testthat
+        run: |
+          Rscript -e 'remotes::install_local("./api/r/cellxgene.census")'
+          Rscript -e 'library("tiledbsoma"); tiledbsoma::show_package_versions()'
+          Rscript -e 'library("testthat"); library("cellxgene.census"); test_dir("./api/r/cellxgene.census/tests/")'
+          Rscript -e 'library("cellxgene.census"); library(testthat); test_file("./api/r/cellxgene.census/tests/testthat/acceptance-tests.R")'
```

Comment:
Pablo created an `acceptance-tests-run-script.R` that does some extra output (for instance adds timestamps, etc). I don't know if this is redundant with GHA, but if might be worth considering.

Reply:
I know about it, but I was unable to get it to work in this context.  Not sure why - seems to be some sort of R sensitivity to current working dir or something like that - so I just cut & pasted the contents of it into this workflow

Reply:
Yeah, I think it needs to be run from the `testthat` directory. I wouldn't worry - we can adjust them later if we want a more detailed output.

Reply:
the other related issue is capturing timings. The python job will print out timings, but R will not. As you said, happy to refine this later.

Reply:
I can take a stab at it later. For now, resolving this.